.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: python
+matrix:
+  include:
+  - python: 2.7
+    env: TOXENV=pep8
+  - python: 2.7
+    env: TOXENV=py27
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+install: pip install tox codecov
+script:
+  - tox
+  - coverage run tests.py
```

Comment:
Overcorps pip installs `codecov` and runs that, as opposed to `coverage`. Now that I'm looking, this might be a typo. Per Chris, to automatically runs coverage.

The other thing we probably want here is to move the coverage checks into `after_success`. Otherwise, codecov will always run, even if the tests failed.